elastic/.gitignore

```diff
@@ -0,0 +1,6 @@
+NOTES.md
```

Comment:
Can we get rid of this like we did on `master`?

Reply:
addressed in latest